commands/metrics/linear/linear-metrics.go

```diff
@@ -50,8 +55,15 @@ var Command = cli.Command{
 		step := ctx.Generic("step")
 		metricsName := ctx.String("name")
 
+		var id *string = nil
+
+		if serviceID := ctx.String("id"); serviceID != "" {
+			id = &serviceID
+		}
```

Comment:
linear metrics can work will service, instance and endpoint too. I think this is not wrong here.